src/Compilers/CSharp/Portable/Syntax/SyntaxNodeRemover.cs

```diff
@@ -37,12 +37,12 @@ internal static class SyntaxNodeRemover
             var residualTrivia = remover.ResidualTrivia;
 
             // the result of the SyntaxRemover will be null when the root node is removed.
```

Comment:
>// the result of the SyntaxRemover will be null when the root node is removed. [](start = 12, length = 78)

Is this comment no longer applicable? Will we get an NRE on dereferencing `result` on line 42?